crates/holochain/tests/inline_zome_sketch.rs

```diff
@@ -34,145 +48,109 @@ fn simple_crud_zome() -> InlineZome {
 
 #[tokio::test(threaded_scheduler)]
 #[cfg(feature = "test_utils")]
-async fn extremely_verbose_inline_zome_sketch() -> anyhow::Result<()> {
+async fn inline_zome_2_agents_1_dna() -> anyhow::Result<()> {
     let envs = test_environments();
 
     // Bundle the single zome into a DnaFile
-
-    let (dna_file, zome) = DnaFile::unique_from_inline_zome("zome1", simple_crud_zome()).await?;
-    let dna_hash = dna_file.dna_hash().clone();
+    let (dna_file, _) = DnaFile::unique_from_inline_zome("zome1", simple_crud_zome()).await?;
 
     // Get two agents
-
-    let (alice, bobbo) = TestAgent::two(envs.keystore()).await;
-    let alice_cell_id = CellId::new(dna_hash.clone(), alice.clone());
-    let bobbo_cell_id = CellId::new(dna_hash.clone(), bobbo.clone());
+    let (alice, bobbo) = TestAgents::two(envs.keystore()).await;
 
     // Create a Conductor
+    let conductor: TestConductorHandle = Conductor::builder().test(&envs).await?.into();
+
+    // Install DNA and install and activate apps in conductor
+    let ids = conductor
+        .setup_app_for_all_agents_with_no_membrane_proof(
+            "app",
+            &[alice.clone(), bobbo.clone()],
+            &[dna_file],
+        )
+        .await;
 
-    let conductor = Conductor::builder().test(&envs).await?;
+    let ((alice,), (bobbo,)) = destructure_test_cells!(ids);
```

Comment:
And here's where we use some amazingly handle itertools magic to destructure the whole thing as tuples of arbitrary arity (up to 4)